src/openrct2-ui/input/Shortcuts.cpp

```diff
@@ -515,6 +517,35 @@ static void TileInspectorMouseDown(WidgetIndex widgetIndex)
     }
 }
 
+static void ShortcutToggleWallSlope()
+{
+    WindowBase* window = WindowFindByClass(WindowClass::TileInspector);
+    if (window == nullptr)
+    {
+        return;
+    }
+
+    const TileElement* tileElement = OpenRCT2::TileInspector::GetSelectedElement();
+
+    // Ensure an element is selected and it's a wall
+    if (tileElement == nullptr || tileElement->GetType() != TileElementType::Wall)
+    {
+        return;
+    }
+
+    int32_t currSlopeValue = tileElement->AsWall()->GetSlope();
+    int32_t newSlopeValue = currSlopeValue + 1;
```

Comment:
You could make this more concise with one line, using the modulus operator:
`int32_t newSlopeValue = (currSlopeValue + 1) % 3;`

Reply:
Resolved, thank you. :)